src/UnitTests/PubnubApi.Tests/WhenAMessageIsPublished.cs

```diff
@@ -31,7 +31,7 @@ public static void Init()
 
             if (!PubnubCommon.PAMEnabled) { return; }
 
-            if (PubnubCommon.PAMEnabled && !string.IsNullOrEmpty(PubnubCommon.SecretKey))
+            if (PubnubCommon.PAMEnabled && string.IsNullOrEmpty(PubnubCommon.SecretKey))
```

Comment:
Out of curiosity was the removal of the `!` intentional here?

Reply:
yes @davidnub  that was intentional change. When Access Manager is enabled and Secret Key is NOT provided, it does not make sense to process in calling Grant() method, else non-200 status code will be returned.

Reply:
Ah ok, thanks for the explanation.

Reply:
cool. If satisfied with the PR, please approve my PR.